src/card/CardList.js

```diff
@@ -13,6 +15,9 @@ export default function CardList({ cards }) {
       {cards.map(card => (
         <Card key={card.id} card={card} />
       ))}
+      <Link to="/create">
```

Comment:
You could also make the Add Button the Link  ( import {Link} from ... into your FormButton Component and then use const Button = styled(Link)`css...` 
This would save you one Element